### PR TITLE
feat: add soil cracking section to site data

### PR DESCRIPTION
**Pull request by paulschreiber**
## Description
Add a soil cracking section to the site data screen.

### Related Issues
Addresses https://github.com/techmatters/terraso-mobile-client/issues/1888

### CODE REVIEW

**Review by tm-ruxandra — CHANGES_REQUESTED** [review]
Requires PO file update

File: dev-client/src/screens/LocationScreens/components/soilId/SiteSoilCracksSection.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      <Heading variant="h6" pt="lg" pb="md">
+        {t('site.soil_id.site_data.cracks.title')}
+      </Heading>
+
+      {soilData.surfaceCracksSelect ? (
+        <Text>
+          {t(`soil.vertical_cracking.value.${soilData.surfaceCracksSelect}`)}
+        </Text>
+      ) : (
+        <></>
```
Comment (knipec) [inline dev-client/src/screens/LocationScreens/components/soilId/SiteSoilCracksSection.tsx:56]: Design nit: The Figma has the spacing consistent regardless of if there's data there or not. This change has them slightly different. Just want to make sure Courtney doesn't mind this?

Screenshot 2024-12-03 at 4 17 27 PM vs Screenshot 2024-12-03 at 4 17 46 PM

**Review by knipec — APPROVED** [review]
Seems good after addressing the comments

**Comment by CourtneyLee333** [comment]
I don't mind. Thanks for checking!

**Pull request merged**